registration-processor/registration-processor-landing-zone/README.md

```diff
@@ -0,0 +1,5 @@
+# Registration Processor landing zone service
+
+## About
+Landing zone performs the following functions:
+*Schedular to moves the packets from dmz server to objectstore if the landing zone type has been changed to objectStore.
```

Comment:
Why do we need this scheduler ? When the landing zone is configured as objectstore, are we not directly uploading to the object store ?  If the design is to uploaded first to landing zone and then schedular moves, what is the frequency of the schedular and what is the recommended the size for the landing zone mount ?
I am hoping this landing zone object store buckets is different from the the one used by the packet manager.

Reply:
As per my knowledge this is to take back up of packet from landing zone dmz server to landing zone object store.
This scheduler will not move packet from landing zone to packet manager bucket .That will take care by packet uploader stage.
This landing zone object store bucket is different that packet manager bucket.

Reply:
So this is line one time utility for someone to move from landzone to objectstore to maintain the packets uploaded from the field ?

Reply:
yeah vishwa I think.

Reply:
Holding this PR for clarification for the below comment.
https://mosip.atlassian.net/browse/MOSIP-20644?focusedCommentId=81748